src/test/java/uk/gov/pay/directdebit/gatewayaccounts/resources/GatewayAccountResourceIT.java

```diff
@@ -420,6 +421,22 @@ public void shouldFailWithBadRequest_whenNoPayload() {
                 .then()
                 .statusCode(Response.Status.BAD_REQUEST.getStatusCode());
     }
+    
+    @Test
+    public void shouldReturnOrganisation_whenItsNotNull() {
+        testGatewayAccount = aGatewayAccountFixture()
+                .withExternalId("osiuoiorga")
+                .withPaymentProvider(PaymentProvider.GOCARDLESS)
+                .withDescription("a org description")
+                .withType(GatewayAccount.Type.TEST)
+                .withAnalyticsId("analytics")
+                .withOrganisation(GoCardlessOrganisationId.valueOf("anorganisat"))
+                .insert(app.getTestContext().getJdbi());
+        givenSetup()
+                .get(GATEWAY_ACCOUNT_API_PATH.replace("{accountId}", "osiuoiorga"))
+                .then()
+                .body("organisation",is("anorganisat"));
+    }
```

Comment:
Maybe also a test checking that the property isn’t there if there’s no organisation?